ui/app/AppLayouts/Wallet/stores/TokensStore.qml

```diff
@@ -9,30 +9,39 @@ QtObject {
     id: root
 
     /* PRIVATE: Modules used to get data from backend */
-    readonly property var _walletModule: walletSectionNewModule
+    readonly property var _allTokensModule: walletSectionAllTokens
     readonly property var _networksModule: networksModule
 
     /* This contains the different sources for the tokens list
        ex. uniswap list, status tokens list */
     readonly property var sourcesOfTokensModel: SortFilterProxyModel {
-        sourceModel: root._walletModule.sourcesOfTokensModel
+        sourceModel: root._allTokensModule.sourcesOfTokensModel
         proxyRoles: ExpressionRole {
             function sourceImage(sourceKey) {
                 return Constants.getSupportedTokenSourceImage(sourceKey)
             }
             name: "image"
             expression: sourceImage(model.key)
         }
+        filters: ExpressionFilter {
+            function filterByKey(sourceKey) {
+                return (sourceKey === Constants.supportedTokenSources.uniswap) ||
+                        (sourceKey === Constants.supportedTokenSources.status)
+            }
+            expression: {
+                return filterByKey(model.key)
+            }
+        }
```

Comment:
Instead of `ExpressionFilter` more efficient combination of value filter could be used:
```suggestion
  filters: AnyOf {
      ValueFilter {
          roleName: "key"
          value: Constants.supportedTokenSources.uniswap
      }
      ValueFilter {
          roleName: "key"
          value: Constants.supportedTokenSources.status
      }
  }
```